frontend/server/cmd/APITool.php

```diff
@@ -1214,6 +1214,30 @@ public function generateDocumentation(): void {
         }
     }
 
+    public function generateControllersDoc(): void {
+        $readmeUrl = 'https://github.com/omegaup/omegaup/blob/main/frontend/server/src/Controllers/README.md';
```

Comment:
Ok, parece que esta no es la mejor forma de agregar la URL, pero tampoco se me ocurren muchas ideas, debido a que este archivo no pasa por el `bootstrap.php`, que es el encargado de cargar todas las constantes.

Así que podemos hacer algo similar a lo que se hace con la constante al inicio del archivo:
```php
define('GITHUB_BASE_URL', 'https://github.com/omegaup/omegaup/blob/main/');
```

Y acá ya la puedes usar así: 
```suggestion
        $readmeUrl = GITHUB_BASE_URL . '/frontend/server/src/Controllers/README.md';
```

Esto en lo que buscamos una mejor forma, quizás una variable de entorno o algo similar.